dll/win32/imm32/keymsg.c

```diff
@@ -1195,3 +1196,89 @@ ImmSendMessageToActiveDefImeWndW(UINT uMsg, WPARAM wParam, LPARAM lParam)
 
     return SendMessageW(hwndIME, uMsg, wParam, lParam);
 }
+
+/***********************************************************************
+ *              ImmCallImeConsoleIME (IMM32.@)
+ */
+DWORD WINAPI
+ImmCallImeConsoleIME(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam, LPUINT puVK)
```

Comment:
Please take the opportunity to use SAL annotations for your functions. (imm32 is fully ours anyway, so we can depart from the wine style.)

Reply:
OK. Done.